### ISSUE TITLE
[DOCS]: Accessibility - add alt text to images in README

### ISSUE THREAD

**Issue by lnovitz** [just-a-job-app/jobseeker-analytics#632]
### Is there an existing request for this change?

- [x] I have searched the existing issues

### Context for documentation change

For folks using text to speech tools, they cannot understand images in docs without alt text

### Collaborators

Thanks Novella for bringing this to my attention 🙏 

### Code of Conduct

- [x] I've read the Code of Conduct and understand my responsibilities as a member of the Just a Job App Community
